mt-challenger/mt_challenger.go

```diff
@@ -4,15 +4,17 @@ import (
 	"context"
 	"time"
 
+	"github.com/urfave/cli"
+
```

Comment:
should delete this line

Reply:
This conforms to the specification of our go format imports